torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1062,6 +1063,77 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
       });
     } break;
 
+    case aten::batch_norm: {
+      bool noWeight = false;
+      bool noBias = false;
+
+      if (v->node()->input(1)->node()->kind() == prim::Constant) {
+        const auto val = toIValue(v->node()->input(1)).value();
+        if (val.isNone()) {
+          noWeight = true;
+        }
+      }
+
+      if (v->node()->input(2)->node()->kind() == prim::Constant) {
+        const auto val = toIValue(v->node()->input(2)).value();
+        if (val.isNone()) {
+          noBias = true;
+        }
+      }
+
+      auto const& shape = valueShape(v->node()->inputs()[0]);
+      return Compute(
+          "aten_batch_norm",
+          c10::fmap<DimArg>(shape),
+          [this, v, noWeight, noBias](const std::vector<VarHandle>& axes) {
+            TORCH_INTERNAL_ASSERT(axes.size() >= 2);
+            auto const& n = v->node();
+
+            // parameter list: input, weight, bias, mean, var, training,
+            // momentum, eps, cudnn_enabled
+            Tensor* input = tensors_.at(n->inputs()[0]->unique());
+            Tensor* mean = tensors_.at(n->inputs()[3]->unique());
+            Tensor* var = tensors_.at(n->inputs()[4]->unique());
+            ExprHandle eps = constant(n->inputs()[7]);
+
+            // axes: N, C, H, W
+            std::vector<VarHandle> c(axes.begin() + 1, axes.begin() + 2);
```

Comment:
You're just getting one axis here, right (c)?  I think I'd be inclined to do `VarHandle c = axes[1];` here instead of constructing a one-element vector.  `call` has a variadic overload so nothing below should have to change (at least, I don't think so)